examples/discovery-search-app/cypress/integration/basic/spec.ts

```diff
@@ -1,12 +1,21 @@
 describe('Basic search', () => {
   beforeEach(() => {
-    cy.server({ force404: true });
+    //TODO: move this section of code into support.js? Maybe just as a helper function
```

Comment:
Looking for feedback on this. This is a large chunk of code that just handles getting the server setup, navigating us to the example app, and mocking out the requests that happen on page load. I could put this into a `beforeEach` in `cypress/support/index.js`, where it would run before *every* test file. That might make it a little less apparent to future developers if they need to override any of these initial routes. Maybe it would be better to define a `loadHomePage()` helper method that does all of this? That would make it easier to see "these are the things we do by default, but you're welcome to do something different."

Reply:
In order to make this work, I had to get typescript working in our cypress tests. I think it might be best to keep those changes isolated from this PR, so I put them in #44, and we can merge it into `develop` after this PR goes in.